image_segmentation/pytorch/runtime/distributed_utils.py

```diff
@@ -0,0 +1,145 @@
+import os
+import random
+
+import torch
+import torch.distributed as dist
+import numpy as np
+
+
+def get_device(local_rank):
+    if torch.cuda.is_available():
+        torch.cuda.set_device(local_rank % torch.cuda.device_count())
+        device = torch.device("cuda")
+    else:
+        device = torch.device("cpu")
+    return device
+
+
+def seed_everything(seed):
+    torch.manual_seed(seed)
+    torch.cuda.manual_seed_all(seed)
```

Comment:
This needs the `if torch.cuda.is_available()` switch.

Reply:
fixed

Reply:
Added a guard, just in case